app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -209,6 +209,15 @@ fun questTypeRegistry(
     element is based on wrong data while the note is not resolved */
     OsmNoteQuestType,
 
+    /* ↓ 0. special case  ------------------------------------------------------------------  */
+    // address: usually only visible when just in front + sometimes requires to take "other answer"
+    // on the other hand it is very useful, and listing it higher show new mappers that SC
+    // asks also about clearly useful data, so someone exploring SC by looking at quest list
+    // will not see merely several quests about bus stops or the bench backrest quest
+    // It is also not very likely to crowd out other quests, as is limited to buildings
```

Comment:
I think that's likely to obscure things related to food and drink establishments (like seating, dietary etc).

Although I do appreciate its an important quest.